modules/ossm-about-adding-projects-using-label-selectors.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/create-mesh.adoc
+
+:_content-type: CONCEPT
+[id="ossm-about-adding-projects-using-label-selectors_{context}"]
+= About adding projects using label selectors
+
+For cluster-wide deployments, you can use label selectors to add projects to the mesh. Label selectors specified in the `ServiceMeshMemberRoll` resource enable the {SMProductShortName} operator to add or remove namespaces to or from the mesh based on namespace labels. Unlike other standard OpenShift resources that allow you to specify a single label selector, the `ServiceMeshMemberRoll` resource allows you to specify multiple label selectors. If the labels for a namespace match any of the selectors specified in the `ServiceMeshMemberRoll` resource, then the namespace is included in the mesh.
+
+[NOTE]
+====
+In {product-title}, a project is essentially a Kubernetes namespace with additional annotations, such as the range of user IDs that can be used in the project. Typically, the {product-title} web console uses the term project, and the CLI uses the term namespace, but the terms are essentially synonymous.
```

Comment:
```suggestion
In {product-title}, a project is identical to a Kubernetes namespace, except with additional annotations, such as the range of user IDs that can be used in the project. Typically, the {product-title} web console uses the term _project_, and the CLI uses the term _namespace_, but the terms are synonymous.
```
- Consider rewording to emphasize that it's a k8s namespace, but with some added characteristics. Your call.
- Per ISG, italicize "Words that are identified as words: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting
- Remove "essentially"; they are either the same or not. If they're not, then tell us what's different.